provisioning/requirements.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - src: arknoll.selenium
   version: 1.2.0
+- src: beetboxvm.upload-progress
+  version: master
```

Comment:
Can you add a release? I've learned from past experience that I need to lock in specific versions for stability reasons—especially with things like PHP extensions which can have problems with new OS/PHP version combos.

Also, since this role doesn't currently support RHEL/CentOS, I would rather not include it since there are a large number of users (myself included) who generally use the centos7 box instead of Ubuntu...